src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java

```diff
@@ -86,6 +87,7 @@ public void should_record_failure_of_upload_once_and_not_reprocess() throws Exce
     }
 
     @Test
+    @Ignore
```

Comment:
why?

Reply:
I am trying to figure out the exact reasons for an error. This is not meant to be merged, not like this at least. That's why I tagged it as wp.

Reply:
Not ignored anymore